diy_asylum_frontend/src/reducers/form.js

```diff
@@ -0,0 +1,55 @@
+// initial state of form
+const initState = {
+  currentStep: 1
+};
+
+// Action Constant Variables
+//
+const INCREMENT_STEP = "INCREMENT_STEP";
+const DECREMENT_STEP = "DECREMENT_STEP";
+
+// Dispatch Actions
+//
+const incrementStep = () => ({
+  type: INCREMENT_STEP
+});
+
+const decrementStep = () => ({
+  type: DECREMENT_STEP
+});
+
+// Action creators, functions will dispatch certain actions
+//
+export const nextFormStep = () => (dispatch, getState) => {
+  const currentStepState = getState().form.currentStep
+  if (currentStepState < 5) {
```

Comment:
This is a little bit of a contrived example of redux-thunk. But the main point of using these "thunk" functions is that when the app starts making async calls to the backend, it's a lot easier to handle Promise logic in these functions than it is elsewhere in the code.

Right now I'm just doing `if` condition checks to illustrate that all sorts of cool logic can happen here 🌈